appveyor.yml

```diff
@@ -4,9 +4,11 @@ environment:
   - DC: dmd
     DVersion: nightly
     arch: x64
+    MODEL: 64
```

Comment:
FWIW this works, because it has no effect. Though fixing this goes over the scope of this PR.

Reply:
What do you mean "it has no effect"?  It's explicitly used when invoking `rdmd_test.exe` and determines how `rdmd` invokes the underlying D compiler.

I'm aware it doesn't affect any part of how the tools are built.

Reply:
Sorry didn't see that line from my phone.

Reply:
OK, no worries.  Just wanted to make sure there was nothing unexpected in how `dmd` on Win64 would react to the `-m` flag.

Reply:
Looks all good:

```
rdmd=generated\windows\32\rdmd.exe -m64 -v  --rdmd-default-compiler=dmd  --test-compilers=dmd
[execute] ["C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\rdmd_app_.exe"]
[execute] ["C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\rdmd_app_.exe", "--help"]3[execute] ["C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\rdmd_app_.exe", "--compiler=dmd", "-m64", "C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\force_src_.d"]
74[execute] 
...
```